Resources/Locale/en-US/accent/italian.ftl

```diff
@@ -109,43 +118,35 @@ accent-italian-words-34 = thing
 accent-italian-words-replace-34 = thing-a
 
 accent-italian-words-35 = traitor
+accent-italian-words-35-2 = traitors
 accent-italian-words-replace-35 = mafioso
 
-accent-italian-words-36 = traitors
-accent-italian-words-replace-36 = mafioso
+accent-italian-words-36 = wine
+accent-italian-words-replace-36 = vino
 
 accent-italian-words-37 = use
 accent-italian-words-replace-37 = use-a
 
 accent-italian-words-38 = want
 accent-italian-words-replace-38 = want-a
 
-accent-italian-words-39 = what's
-accent-italian-words-replace-39 = what's-a
+accent-italian-words-39 = am
+accent-italian-words-replace-39 = am-a
 
-accent-italian-words-40 = who's
-accent-italian-words-replace-40 = who's-a
+accent-italian-words-40 = and
+accent-italian-words-replace-40 = and-a
 
 accent-italian-words-41 = whose
 accent-italian-words-replace-41 = whose-a
 
 accent-italian-words-42 = why
 accent-italian-words-replace-42 = for-a what reason
 
-accent-italian-words-43 = wine
-accent-italian-words-replace-43 = vino
+accent-italian-words-43 = passenger
+accent-italian-words-replace-43 = goombah
 
-accent-italian-words-44 = passenger
-accent-italian-words-replace-44 = goombah
+accent-italian-words-44 = passengers
+accent-italian-words-replace-44 = goombahs
 
-accent-italian-words-45 = passengers
-accent-italian-words-replace-45 = goombahs
-
-accent-italian-words-46 = i'm
-accent-italian-words-replace-46 = i'm-a
-
-accent-italian-words-47 = am-a
-accent-italian-words-replace-47 = am-a
-
-accent-italian-words-48 = and-a
-accent-italian-words-replace-48 = and-a
+accent-italian-words-45 = meat
+accent-italian-words-replace-45 = prosciutto
```

Comment:
Doesn't this duplicate the replacement in the new accent system? I would rather you keep this but remove the custom accent system.

Reply:
this is to change all meat to prosciutto, so the small part in the accent system can change it to pepperoni 50% of the time